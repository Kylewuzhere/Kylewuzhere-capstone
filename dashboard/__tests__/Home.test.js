import renderer from "react-test-renderer";
import Home from "../src/app/page.js";

describe("Home", () => {
  it("renders Home correctly", async () => {
    const tree = renderer.create(<Home />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
