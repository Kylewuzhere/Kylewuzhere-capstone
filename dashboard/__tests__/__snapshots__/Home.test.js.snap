// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders Home correctly 1`] = `
<div
  className="flex min-h-screen pb-24 items-center justify-center bg-grey-dark"
>
  <button
    className="text-white bg-orange py-2 px-7 rounded text-sm"
    onClick={[Function]}
  >
    Sign in
  </button>
</div>
`;
